gspread_formatting/models.py

```diff
@@ -187,11 +189,29 @@ def __init__(self, themeColor=None, rgbColor=None):
 class Color(CellFormatComponent):
     _FIELDS = ('red', 'green', 'blue', 'alpha')
 
-    def __init__(self, red=None, green=None, blue=None, alpha=None):
+    def __init__(self, red=None, green=None, blue=None, alpha=None, hexcolor=None):
         self.red = red
         self.green = green
         self.blue = blue
         self.alpha = alpha
+        if hexcolor:
+            self.fromHex(hexcolor)
+
+    def toHex(self):
+        # Dont worry about alpha channel yet
+        # This would simpler if the default was 0 instead of None
+        RR = format(floor((self.red if self.red else 0) * 255), '02x')
```

Comment:
You won't need `math.floor`, since `int` itself will truncate toward zero and that's sufficient for color values, which are always non-negative.